include/quic/gnutls_crypto.hpp

```diff
@@ -23,6 +25,12 @@ namespace oxen::quic
             unsigned int incoming,
             const gnutls_datum_t* msg)>;
 
+    constexpr auto GNUTLS_KEY_SIZE = 32;  // for now, only supporting Ed25519 keys (32 bytes)
```

Comment:
This might be better as an explicit `size_t` rather than auto (which makes it an `int`).

Reply:
makes sense